doc/userguide/rules/vlan-keywords.rst

```diff
@@ -0,0 +1,87 @@
+VLAN Keywords
+=============
+
+.. role:: example-rule-action
+.. role:: example-rule-header
+.. role:: example-rule-options
+.. role:: example-rule-emphasis
+
+vlan.id
+-------
+
+Suricata has a ``vlan.id`` keyword that can be used in signatures to identify
+and filter network packets based on Virtual Local Area Network IDs. By default,
+it matches all layers if a packet contains multiple VLAN layers. However, if a
+specific layer is defined, it will only match that layer.
+
+This keyword supports ``all`` and ``count`` as arguments for ``layer``.
+``all`` matches only if all VLAN layers match and ``count`` matches based on the number of layers.
+
+Id values for vlan.id keyword:
+
+========  ================================================
+Value     Description
+========  ================================================
+1 - 4094  Valid range for vlan id
+0 - 3     Valid range of number of layers (with ``count``)
+========  ================================================
```

Comment:
We can use the `.. table:: ` Sphinx directive and have neat table titles, here and for the following one:
```suggestion
.. table:: **Id values for vlan.id keyword**

    ========  ================================================
    Value     Description
    ========  ================================================
    1 - 4094  Valid range for vlan id
    0 - 3     Valid range of number of layers (with ``count``)
    ========  ================================================
```